contracts/HydraDelegation/Delegation.sol

```diff
@@ -111,23 +111,25 @@ contract Delegation is
     function getRawDelegatorReward(address staker, address delegator) external view returns (uint256) {
         uint256 reward = getRawReward(staker, delegator);
         uint256 commission = delegationCommissionPerStaker[staker];
-        if (commission == 0) return reward;
+        if (commission != 0) {
+            (, reward) = _applyCommission(reward, commission);
+        }
 
-        (, uint256 rawDelegatorReward) = _applyCommission(reward, commission);
-        return rawDelegatorReward;
+        return reward;
     }
 
     /**
      * @inheritdoc IDelegation
      */
     function getDelegatorReward(address staker, address delegator) external view returns (uint256) {
-        uint256 reward = getRawReward(staker, delegator);
-        uint256 aprReward = aprCalculatorContract.applyBaseAPR(reward);
+        uint256 rawReward = getRawReward(staker, delegator);
+        uint256 reward = aprCalculatorContract.applyBaseAPR(rawReward);
```

Comment:
Since adding an additional if to return early, if a reward is 0 might be beneficial. In most cases we expect to call this function when the reward is not 0. Also the commission calculations are minimal so changing it to return early would have minimal effect only when it is 0.